packages/pagination/src/PaginationControls.js

```diff
@@ -73,30 +72,29 @@ const PaginationControls = ({
       }
 
       let breakView;
-
-      pages.forEach((pageNumber, index) => {
+      let index;
+      let pageNumber;
+      for (index = 0; index < pageCount; index++) {
```

Comment:
Since `index` is only used in this scope, I would define it in the `for statement`, ie `let index = 0`, instead of above.

Reply:
updated in #78 

https://github.com/Availity/availity-react/pull/78/commits/ea3d0e4331c813d8c85608ce50533ac58f42fd18#diff-912d664f4d49898b10ffaac011b8cb5eR78